src/web/client/dal/remoteSaveProvider.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
+import fetch, { RequestInit } from "node-fetch";
```

Comment:
@tyaginidhi , Vipin has used node-fetch instead of vanilla JS fetch method. Few reasons:
1. It is a light weight NPM package which is similar to JS fetch.
2. JS fetch mocking was not achievable so need to switch to node-fetch.